source/iface/host_accessor.rst

```diff
@@ -2,8 +2,656 @@
+
+  template <typename AllocatorT, typename TagT>
+  host_accessor(buffer<DataT, Dimensions, AllocatorT>& bufferRef,
+                range<Dimensions> accessRange, id<Dimensions> accessOffset,
+                TagT tag, const property_list& propList = {})
+
+Constructors of the host_accessor class.
+The descriptions are listed in |SYCL_ACCESS_HOST_CON|.
+
+Member functions
+================
+
+``swap``
+========
+
+::
+
+  void swap(host_accessor& other);
+
+Swaps the contents of the current accessor with
+the contents of ``other``.
+
+``get_offset``
+==============
+
+::
+
+  id<Dimensions> get_offset() const
+
+Available only when ``(Dimensions > 0)``.
+
+If this is a ranged accessor, returns the offset that was specified
+when the accessor was constructed. For other accessors,
+returns the default constructed ``id<Dimensions>{}``.
+
+``get_pointer``
+===============
+
+::
+
+  std::add_pointer_t<value_type> get_pointer() const noexcept
+
+Returns a pointer to the start of this accessor’s underlying buffer,
+even if this is a ranged accessor whose range does not start at the
+beginning of the buffer. The return value is unspecified
+if the accessor is empty.
+
+``operator=``
+=============
+
+::
+
+  const host_accessor& operator=(const value_type& other) const
+
+Available only when ``(AccessMode != access_mode::read && Dimensions == 0)``.
+
+Assignment to the single element that is accessed by this accessor.
+
+::
+
+  const host_accessor& operator=(value_type&& other) const
+
+Available only when ``(AccessMode != access_mode::read && Dimensions == 0)``.
+
+Assignment to the single element that is accessed by this accessor.
+
+
+.. _deduction_tags_buff_host_accessors:
+
+=========================
+Buffer host accessor tags
+=========================
+
+Some ``host_accessor`` constructors take a ``TagT`` parameter,
+which is used to deduce template arguments.
+The permissible values for this parameter are listed in table below.
+
+.. list-table::
+  :header-rows: 1
+
+  * - Tag value
+    - Access mode
+  * - ``read_write;``
+    - ``access_mode::read_write``
+  * - ``read_only;``
+    - ``access_mode::read``
+  * - ``write_only;``
+    - ``access_mode::write``
+
+``read-only host accessors``
+============================
+
+.. list-table::
+  :header-rows: 1
+
+  * - Data type
+    - Access mode
+  * - ``not const-qualified``
+    - ``access_mode::read``
+  * - ``const-qualified``
+    - ``access_mode::read``
+
+The specializations of ``host_accessor``
+that are read-only accessors.
+There is an implicit conversion between any of these specializations,
+provided that all other template parameters are the same.
+
+``read-write host accessors``
+=============================
+
+.. list-table::
+  :header-rows: 1
+
+  * - Data type
+    - Access mode
+  * - ``not const-qualified``
+    - ``access_mode::read_write``
+
+The table present an implicit conversion from
+the read-write specialization.
```

Comment:
I don't think the table presents that, so this wording is not correct.  The spec text following relates two tables, which is different from the wording here proposed for the reference.

> There is also an implicit conversion from the read-write host_accessor type shown in [Table 75](https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#table.accessors.host.buffer.read-write) to any of the read-only accessors in [Table 74](https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#table.accessors.host.buffer.read-only), provided that all other template parameters are the same.

Reply:
Please check if this context present correct state: https://github.com/KhronosGroup/SYCL_Reference/pull/98/commits/0a37df862ec9016a9e55cb029c008f842e1d09ab